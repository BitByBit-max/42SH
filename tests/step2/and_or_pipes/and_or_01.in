echo "I could've" || echo error && echo dropped || echo  error && echo my || echo error && echo croissant!
