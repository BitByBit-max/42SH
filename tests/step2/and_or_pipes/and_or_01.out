I could've
dropped
my
croissant!
